pkg/stub/handler.go

```diff
@@ -430,24 +430,25 @@ func (h *Handler) tbrInaccessible() bool {
 		h.imageConfigBlocksImageStreamCreation("quay.io") {
 		return true
 	}
-	err := wait.PollImmediate(5*time.Second, 20*time.Second, func() (bool, error) {
+
+	proxy := &configv1.Proxy{}
+	var err error
```

Comment:
nit: The error from inside of the `wait.pollImmediate` is never surfaced outside of that function, so this probably isn't needed here (but is for the second one below)

Reply:
actually no with what I am doing with proxy I either need to do this as I have it, or I have to rename the var

leaving this as is